soh/soh/Enhancements/randomizer/entrance.cpp

```diff
@@ -367,6 +367,33 @@ void SetAllEntrancesData() {
         { { EntranceType::SpecialInterior, RR_KAK_BACKYARD,          RR_KAK_POTION_SHOP_BACK,  ENTR_POTION_SHOP_KAKARIKO_BACK },
           { EntranceType::SpecialInterior, RR_KAK_POTION_SHOP_BACK,  RR_KAK_BACKYARD,          ENTR_KAKARIKO_VILLAGE_OUTSIDE_POTION_SHOP_BACK } },
         
+        { { EntranceType::GerudoFortress, RR_GF_OUTSKIRTS, RR_TH_1_TORCH_CELL, ENTR_THIEVES_HIDEOUT_0 },
+          { EntranceType::GerudoFortress, RR_TH_1_TORCH_CELL, RR_GF_OUTSKIRTS, ENTR_GERUDOS_FORTRESS_1 } },
+        { { EntranceType::GerudoFortress, RR_GF_NEAR_GROTTO, RR_TH_1_TORCH_CELL, ENTR_THIEVES_HIDEOUT_1 },
+          { EntranceType::GerudoFortress, RR_TH_1_TORCH_CELL, RR_GF_NEAR_GROTTO, ENTR_GERUDOS_FORTRESS_2 } },
+        { { EntranceType::GerudoFortress, RR_GF_NEAR_GROTTO, RR_TH_KITCHEN_CORRIDOR, ENTR_THIEVES_HIDEOUT_2 },
+          { EntranceType::GerudoFortress, RR_TH_KITCHEN_CORRIDOR, RR_GF_NEAR_GROTTO, ENTR_GERUDOS_FORTRESS_3 } },
+        { { EntranceType::GerudoFortress, RR_GF_ABOVE_GTG, RR_TH_KITCHEN_CORRIDOR, ENTR_THIEVES_HIDEOUT_3 },
+          { EntranceType::GerudoFortress, RR_TH_KITCHEN_CORRIDOR, RR_GF_ABOVE_GTG, ENTR_GERUDOS_FORTRESS_4 } },
+        { { EntranceType::GerudoFortress, RR_GF_NEAR_GROTTO, RR_TH_STEEP_SLOPE_CELL, ENTR_THIEVES_HIDEOUT_4 },
+          { EntranceType::GerudoFortress, RR_TH_STEEP_SLOPE_CELL, RR_GF_NEAR_GROTTO, ENTR_GERUDOS_FORTRESS_5 } },
+        { { EntranceType::GerudoFortress, RR_GF_BOTTOM_OF_LOWER_VINES, RR_TH_STEEP_SLOPE_CELL, ENTR_THIEVES_HIDEOUT_5 },
+          { EntranceType::GerudoFortress, RR_TH_STEEP_SLOPE_CELL, RR_GF_BOTTOM_OF_LOWER_VINES, ENTR_GERUDOS_FORTRESS_6 } },
+        { { EntranceType::GerudoFortress, RR_GF_ABOVE_GTG, RR_TH_DOUBLE_CELL, ENTR_THIEVES_HIDEOUT_6 },
+          { EntranceType::GerudoFortress, RR_TH_DOUBLE_CELL, RR_GF_ABOVE_GTG, ENTR_GERUDOS_FORTRESS_7 } },
+        { { EntranceType::GerudoFortress, RR_GF_TOP_OF_LOWER_VINES, RR_TH_DOUBLE_CELL, ENTR_THIEVES_HIDEOUT_7 },
+          { EntranceType::GerudoFortress, RR_TH_DOUBLE_CELL, RR_GF_TOP_OF_LOWER_VINES, ENTR_GERUDOS_FORTRESS_8 } },
+        { { EntranceType::GerudoFortress, RR_GF_TOP_OF_LOWER_VINES, RR_TH_KITCHEN_TOP, ENTR_THIEVES_HIDEOUT_8 },
```

Comment:
RR_KITCHEN_TOP should be split to allow decoupled logic to route through without card